client/components/HostAndRooms.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const HostAndRooms = ({ hostAndRooms }) => {
+  let summary = '';
```

Comment:
I like this!  separate from props from the get go.

Reply:
Yeahh, destructuring is the way to go! Who wants to write props all the time???